src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/people`} passHref>
-              <IconButton
-                aria-label="People"
-                className={classes.roundButton}
-                color={key.startsWith('/people') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <People />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/journeys`} passHref>
-              <IconButton
-                aria-label="Journeys"
-                className={classes.roundButton}
-                color={key.startsWith('/journeys') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <Explore />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/areas`} passHref>
-              <IconButton
-                aria-label="Areas"
-                className={classes.roundButton}
-                color={key.startsWith('/areas') ? 'primary' : 'secondary'}
-                data-test="area-button"
-                size="large"
-              >
-                <Map />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/projects`} passHref>
-              <IconButton
-                aria-label="Projects"
-                className={classes.roundButton}
-                color={key.startsWith('/projects') ? 'primary' : 'secondary'}
-                data-test="calendar-button"
-                size="large"
+  const [open, setOpen] = useState(false);
+  const toggleDrawer = () => {
+    setOpen(!open);
+    setExpanded(!expanded);
+  };
+
+  const model = useModel((env) => new OrganizationsDataModel(env));
+
+  const menuItemsMap = [
+    { icon: <Map />, name: 'areas' },
+    { icon: <Home />, name: 'home' },
+    { icon: <Explore />, name: 'journeys' },
+    { icon: <People />, name: 'people' },
+    { icon: <Event />, name: 'projects' },
+  ] as const;
+
+  const OrganizeSideBar = (
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
+      <Drawer
+        onFocus={() => void 0}
+        onMouseLeave={() => {
+          setHover(false);
+        }}
+        onMouseOver={() => {
+          setHover(true);
+        }}
+        open={open}
+        sx={{
+          alignItems: open ? 'flex-start' : 'center',
+        }}
+        variant="permanent"
+      >
+        <DrawerHeader>
+          <IconButton onClick={toggleDrawer}>
+            {!open && hover && !expanded && (
+              <KeyboardDoubleArrowRightIcon
+                onClick={toggleDrawer}
```

Comment:
You already have a click on the button, so you shouldn't need another click on the icon.